build.sh

```diff
@@ -74,9 +80,12 @@ function printHelp() {
     {
         printf " \t%s\t%s\n" "--test" "to run the tests"
         printf " \t%s\t%s\n" "--testUi" "to run the tests including UI tests"
+        printf " \t%s\t%s\n" "--installCore" "install JMC core"
         printf " \t%s\t%s\n" "--packageJmc" "to package JMC"
+        printf " \t%s\t%s\n" "--packageAgent" "to package Agent"
         printf " \t%s\t%s\n" "--clean" "to run maven clean"
         printf " \t%s\t%s\n" "--run" "to run JMC once it was packaged"
+        printf " \t%s\t%s\n" "--runAgentExample" "to run Agent Example once it was packaged"
```

Comment:
Any chance you could also add --runAgentConverterExample? :)

Basically:
--add-opens java.base/jdk.internal.misc=ALL-UNNAMED -XX:+FlightRecorder -javaagent:${resource_loc:/org.openjdk.jmc.agent/target/org.openjdk.jmc.agent-1.0.0-SNAPSHOT.jar}=${resource_loc:/org.openjdk.jmc.agent/src/test/resources/org/openjdk/jmc/agent/converters/test/jfrprobes_template.xml}

Reply:
Although I'm not an reviewer 😎 it seems that the `build.cmd` is missing the `--runAgentExample` additon mentioned by @thegreystone. Anything else looks ok.

Reply:
the reason of missing --runAgentExample option is the proper java version detection. I've not found quite nice way hot to do it. Maybe you have some proposal @reinhapa  ?

Reply:
I've add an option --runAgentCustomExample '<class>' => --runAgentCustomExample 'org.openjdk.jmc.agent.test.InstrumentMe' , what do you think ? @thegreystone

Reply:
For me, the buildscript is all about making it easier to do common things. If --runAgentConvertExample makes it easy to run one of the two examples one might want to run to test things from JMC. If I need to do something custom, I might as well just put the built agent jar on the command line and do whatever I want to do. Since the classpath won't be set to anything else, there won't be that many main classes to pick from other than the standard example and the converter example. 

I would suggest just having --runAgentExample and --runAgentConverterExample.

Reply:
agree, updated 2 options: --runAgentExample and --runAgentConverterExample.